### PR TITLE
Deduplicate code in the Pregnancy Stats screen. 

### PR DESCRIPTION
**Pull request by uglybead**
### Things to include in a large Pull Request:

- What is the purpose of the pull request?

Deduplicate code in the Pregnancy Stats screen. Also adds a statistic for percentage of offspring met.

- Give a brief description of what you changed or added.

As in the purpose.

- Are any new graphical assets required?

No.

- Has this change been tested? If so, mention the version number that the test was based on.

Yes. Works at head.

- So we have a better idea of who you are, what is your Discord Handle?

n/a

- If you want quick feedback, you can use @Innoxia, or jump on the Lilith's Throne discord and send Innoxia a PM.

### CODE REVIEW

File: src/com/lilithsthrone/game/dialogue/utils/PhoneDialogue.java [hunk 1]
```diff
@@ -997,11 +997,61 @@ public DialogueNodeType getDialogueNodeType() {
 	public static final DialogueNodeOld CHARACTER_STATS_PREGNANCY = new DialogueNodeOld("Pregnancy Stats", "", true) {
 		private static final long serialVersionUID = 1L;
 
+		private void OffspringHeaderDisplay(StringBuilder output, String word_one, String word_two, String color, int count) {
+			output.append("<div class='extraAttribute-quarter'>");
+			output.append(word_one);
+			output.append("<br/>");
```
Comment (picobyte) [inline src/com/lilithsthrone/game/dialogue/utils/PhoneDialogue.java]: shouldn't above line be `output.append("<br>");`? it was like this in the original code, but I think that may be wrong?

Comment (CognitiveMist) [reply src/com/lilithsthrone/game/dialogue/utils/PhoneDialogue.java]: `br` is a self-closing tag (and a bit of a special case). Technically, `<br/>` is most correct, but historically `<br>` was used most often. Nearly all html renderers will treat both the same way. An actual closing tag would look like `</br>`, and would almost always be incorrect.

Comment (uglybead) [reply src/com/lilithsthrone/game/dialogue/utils/PhoneDialogue.java]: In the existing codebase:

* the "<br>" form is used once (in com/lilithsthrone/game/dialogue/utils/OptionsDialogue.java)
* the "<br/>" form is used at least 2574 times (number of lines where it occurs)
* the "<br />" form is used zero times

It seems like what is done here aligns most with the existing codebase.

**Comment by Innoxia** [comment]
Thank you for this!

**Pull request merged**